src/NzbDrone.Core/IndexerSearch/Definitions/SearchCriteriaBase.cs

```diff
@@ -19,6 +20,9 @@ public abstract class SearchCriteriaBase
         public virtual bool MonitoredEpisodesOnly { get; set; }
         public virtual bool UserInvokedSearch { get; set; }
 
+        public Artist Artist { get; set; }
+        public List<Track> Tracks { get; set; }
```

Comment:
Do we need to add albums as well since we will be search by album when searching online in most cases?